pkg/controller/provider/xavier/vsphere/adapter.go

```diff
@@ -0,0 +1,78 @@
+package vsphere
+
+import (
+	liberr "github.com/konveyor/controller/pkg/error"
+	libmodel "github.com/konveyor/controller/pkg/inventory/model"
+	model "github.com/konveyor/virt-controller/pkg/controller/provider/model/vsphere"
+	"github.com/konveyor/virt-controller/pkg/controller/provider/xavier/base"
+)
+
+//
+// Model adapter.
+type Adapter struct {
+	// DB
+	libmodel.DB
+}
+
+//
+// Update the vSphere xavier body.
+func (r *Adapter) UpdateBody(object base.Object, m libmodel.Model) (err error) {
```

Comment:
This returns an error, but its only caller ignores the error. Is that an oversight, or deliberate?

Reply:
Oversight.  Fixed.